src/sdk/account/utils/toSigner.ts

```diff
@@ -5,24 +5,36 @@ import {
   type EIP1193Provider,
   type EIP1193RequestFn,
   type EIP1474Methods,
+  type Hex,
   type LocalAccount,
   type OneOf,
   type Transport,
   type WalletClient,
   createWalletClient,
-  custom
+  custom,
+  getAddress,
+  hexToBytes
 } from "viem"
 import { toAccount } from "viem/accounts"
 
 import { signTypedData } from "viem/actions"
 import { getAction } from "viem/utils"
 
+// // @Review
+// // This type helps providing other signer types
+// type MinimalSigner = {
+//   signMessage(message: any): Promise<string>
+//   signTypedData(domain: any, types: any, value: any): Promise<string>
+//   [key: string]: any
+// }
+
 export type Signer = LocalAccount
 export type UnknownSigner = OneOf<
   | EIP1193Provider
   | WalletClient<Transport, Chain | undefined, Account>
   | LocalAccount
   | Account
+  | any
```

Comment:
Don't think there's any utility to having the previous types if you add `any` to the end of the list. It might as well be `export type UnknownSigner = any`